### PR TITLE
[Xcodeproj] Add "@loader_path/Frameworks" to `LD_RUNPATH_SEARCH_PATHS` in test target

### PR DESCRIPTION
**Pull request by norio-nomura**
Fix https://bugs.swift.org/browse/SR-3913

### CODE REVIEW

**Comment by aciidb0mb3r** [comment]
@abertelrud Can you review this?

File: Sources/Xcodeproj/pbxproj().swift [hunk 1]
```diff
@@ -358,7 +358,7 @@ func xcodeProject(
         
         if module.isTest {
             targetSettings.common.EMBEDDED_CONTENT_CONTAINS_SWIFT = "YES"
-            targetSettings.common.LD_RUNPATH_SEARCH_PATHS = ["@loader_path/../Frameworks"]
+            targetSettings.common.LD_RUNPATH_SEARCH_PATHS = ["@loader_path/../Frameworks", "@loader_path/Frameworks"]
```
Comment (aciidb0mb3r) [inline Sources/Xcodeproj/pbxproj().swift]: Please add a test

Comment (norio-nomura) [inline Sources/Xcodeproj/pbxproj().swift]: Updated test. 🙇

Comment (aciidb0mb3r) [inline Sources/Xcodeproj/pbxproj().swift]: thanks!

**Comment by abertelrud** [comment]
Looks good to me.  It's unfortunate that all products will end up with both search path, when only one of them is ever used, but it does no harm.

**Comment by aciidb0mb3r** [comment]
@norio-nomura Can you also raise this PR on master branch?

**Comment by rballard** [comment]
Approved for the Swift 3.1 branch.

**Comment by aciidb0mb3r** [comment]
@swift-ci please test

**Comment by aciidb0mb3r** [comment]
@swift-ci please test

**Comment by norio-nomura** [comment]
Thank you! 🙏 
I opened #950 against master.

**Pull request merged**